src/views/layouts/default.njk

```diff
@@ -34,7 +34,7 @@
   {% include "partials/nav/top.njk" %}
   {% if backURL %}
     {{ govukBackLink({
-    href: backURL + "?lang=" +lang,
+    href: backURL +"?companyNumber="+ companyNumber + "&"+ "selectedPscId="+ selectedPscId +"%3F",
```

Comment:
maybe for extensions these params are needed for every page, but I wonder if it is good to expect that?  
In PSC verification we have [addSearchParams](https://github.com/companieshouse/psc-verification-web/blob/33dea3d747ae5ff93f39ce28e4775be7565956a8/src/utils/queryParams.ts#L1) which produces this.  But the url in the verification journey does vary more than in the extensions one, so mentioning for consideration.